src/gucs.rs

```diff
@@ -7,13 +7,29 @@ All rights reserved.
 Use of this source code is governed by the PostgreSQL license that can be found in the LICENSE.md file.
 */
 
+use once_cell::sync::Lazy;
 use pgx::guc::{GucContext, GucRegistry, GucSetting};
 use std::path::PathBuf;
 use std::str::FromStr;
 
 static PLRUST_WORK_DIR: GucSetting<Option<&'static str>> = GucSetting::new(None);
 static PLRUST_PG_CONFIG: GucSetting<Option<&'static str>> = GucSetting::new(None);
 static PLRUST_TRACING_LEVEL: GucSetting<Option<&'static str>> = GucSetting::new(None);
+static PLRUST_ALLOWED_DEPENDENCIES: GucSetting<Option<&'static str>> = GucSetting::new(None);
+
+static PLRUST_ALLOWED_DEPENDENCIES_CONTENTS: Lazy<toml::value::Table> = Lazy::new(|| {
+    let path = PathBuf::from_str(
```

Comment:
Kinda wonder if I can make the GucSettings hold `&'static Path`... I'll look into that later.